src/reducers/articles.js

```diff
@@ -0,0 +1,20 @@
+import {
+  GET_ALL_ARTICLES,
+  GET_ALL_ARTICLES_SUCCESS_MSG,
+  GET_ALL_ARTICLES_FAILURE_MSG,
+} from '../actions/types';
+
+const initialState = { articles: [], message: null };
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case GET_ALL_ARTICLES:
+      return { ...state, articles: action.payload };
+    case GET_ALL_ARTICLES_SUCCESS_MSG:
+      return { ...state, message: action.payload };
+    case GET_ALL_ARTICLES_FAILURE_MSG:
+      return { ...state, message: action.payload };
+    default:
+      return state;
```

Comment:
Thank you for now mutating the state here.

Reply:
@frostyblok, I did not mutate state here 😆